csharp/Platform.IO/StreamExtensions.cs

```diff
@@ -16,6 +33,22 @@ public static void Write<T>(this Stream stream, T value)
             stream.Write(bytes, 0, bytes.Length);
         }
 
+        /// <summary>
+        /// <para>Reads a sequence that represents the <typeparamref name="T"/> structure and moves the current position of the <paramref name="stream"/> by the number of read bytes.</para>
+        /// <para>Считывает последовательность байт представляющих структуру типа <typeparamref name="T"/> и перемещает текущую позицию в <paramref name="stream"/> вперед на число прочитанных байт.</para>
+        /// </summary>
+        /// <typeparam name="T">
+        /// <para>The structure type.</para>
+        /// <para>Тип являющийся структурой.</para>
+        /// </typeparam>
+        /// <param name="stream">
+        /// <para>A stream containing the <typeparam name="T"> structure.</para>
+        /// <para>Поток, содержащий структуру <typeparam name="T">.</para>
+        /// </param>
+        /// <returns>
+        /// <para>The <typeparam name="T"> structure, if its bytes from<paramref name="stream"/> have been read, otherwise the default value of <typeparamref name="T"/> structure.</para>
```

Comment:
`if its bytes from<paramref name="stream"/> have been read` ⇾ `if its bytes from the <paramref name="stream"/> are read`

Reply:
`The <typeparam name="T"> structure` ⇾ `The <typeparam name="T"> structure value`